canister/src/api/get_balance.rs

```diff
@@ -151,7 +153,8 @@ mod test {
                 get_balance(GetBalanceRequest {
                     address: address.to_string(),
                     min_confirmations: *min_confirmations
-                }),
+                })
+                .unwrap(),
                 1000
```

Comment:
nit: instead of unwrapping, I suggest we compare with `Ok(_)`, which returns an easier to read error if the assertion fails. Ditto everywhere else.

```suggestion
                }),
                Ok(1000)
```